dcpy/library/script/fisa_dailybudget.py

```diff
@@ -72,6 +79,17 @@ def ingest(self) -> pd.DataFrame:
         df = self._dedupe(df)
         return df.sort_values(by="TBL_LAST_DT")
 
+    def ingest(self) -> pd.DataFrame:
```

Comment:
since this one uses `previous_version`, should this be `ingest_historical`?

Reply:
This is meant to be just the standard ingest. Ingest historical was meant if you had the entire historical archive of files to process. Maybe not the clearest lingo

Reply:
I can add some helpful comments/docstrings

Reply:
gotcha. and looks like `ingest_historical` isn't called in `runner`, so someone would have to edit this file to use `ingest_historical`?

Reply:
And by that I mean delete the now-defunct function